pkg/services/auth/auth_token.go

```diff
@@ -133,6 +141,13 @@ func (s *UserAuthTokenService) LookupToken(ctx context.Context, unhashedToken st
 		return nil, models.ErrUserTokenNotFound
 	}
 
+	if model.CreatedAt <= s.createdAfterParam() || model.RotatedAt <= s.rotatedAfterParam() {
```

Comment:
This changes the logic compared to earlier which uses greater than compared to greater than or equal and in addition AND instead of OR. Any reason for this change?

Reply:
Yes, we want the opposite of what were done before. Before, we wanted to only retrieve valid tokens so the condition was equal to "token is not expired". Now, we want to retrieve valid and expired tokens but if the token is expired we return an error, so the condition here is "token is expired".

Reply:
Yeah of course. Sorry for the confusion